docs/legacy/porting-guide.md

```diff
@@ -246,8 +246,8 @@ The v8 luma.gl API was designed to allow apps to work directly with the `WebGLRe
 
 | v8 Prop or Method    | v9 Replacement                                      | Comment                                                                                                                             |
 | -------------------- | --------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------- |
-| `makeDebugContext()` | `luma.createDevice({debug: true, type: 'webgl'})`   | [Khronos WebGL developer tools][khronos_dev_tools] are dynamically loaded when needed.                                              |
-| Spector.js           | `luma.createDevice({spector: true, type: 'webgl'})` | [Spector.js][spector] is pre-integrated. Te Spector.js library will be dynamically loaded when needed and the canvas is "captured". |
+| `makeDebugContext()` | `luma.createDevice({webgl: {debug: true}, type: 'webgl'})` | [Khronos WebGL developer tools][khronos_dev_tools] are dynamically loaded when needed.                                              |
+| Spector.js           | `luma.createDevice({webgl: {debugWithSpectorJS: true}, type: 'webgl'})` | [Spector.js][spector] is pre-integrated. The Spector.js library will be dynamically loaded when needed and the canvas is "captured". |
```

Comment:
Ditto, except this was originally a typo and would be "debugWithSpectorJS"

Reply:
> Ditto, except this was originally a typo and would be "debugWithSpectorJS"

Fixed thanks